framework/source/class/qx/log/Logger.js

```diff
@@ -578,9 +578,17 @@ qx.Bootstrap.define("qx.log.Logger",
       // Add relation fields
       if (object)
       {
+        function isQxCoreObject(clz) {
+          while (clz) {
+            if (clz.classname === "qx.core.Object")
+              return true;
```

Comment:
You'll need to use braces to avoid linting errors: https://travis-ci.org/qooxdoo/qooxdoo/jobs/591902970#L542